### ISSUE TITLE
bats tests should verify that packages are signed

### ISSUE THREAD

**Issue by nik9000** [elastic/elasticsearch#13182]
The bats tests currently simply install the rpm and debs an exercise them. They should also check that they are signed.

This is part of #13138.

**Comment by s1monw** [comment]
@nik9000 I am not sure we can easily do this our bulids don't sign the builds at all? maybe they should?

**Comment by nik9000** [comment]
> @nik9000 I am not sure we can easily do this our bulids don't sign the builds at all? maybe they should?

Something like that, yeah. Maybe some self signed certificate stuff. We can figure it out.

**Comment by s1monw** [comment]
yeah I wonder if the release script and the smoke tester should do it for now at least?

**Comment by nik9000** [comment]
> yeah I wonder if the release script and the smoke tester should do it for now at least?

Makes sense to me.

**Comment by nik9000** [comment]
Well that was a trip!
https://github.com/mojohaus/rpm-maven-plugin/issues/11
and
rpm from homebrew is a bit wonky and needs love:
```shell
$ cat ~/.rpmmacros 
%__gpg_check_password_cmd       %{__gpg} \
        gpg --homedir %{_gpg_path} --batch --no-verbose --passphrase-fd 3 -u %{_gpg_name} -so -
__gpg_sign_cmd                 %{__gpg} \
        gpg --homedir %{_gpg_path} --batch --no-verbose --no-armor --passphrase-fd 3 --no-secmem-warning \
        -u %{_gpg_name} -sbo %{__signature_filename} %{__plaintext_filename}
```

**Comment by clintongormley** [comment]
Closed by #13354
